filevalidator.js

```diff
@@ -11,7 +11,13 @@ function(_) {
             // MPEG-1 Layer 3 file without an ID3 tag or with an ID3v1 tag (which's appended at the end of the file)
             [0xFF, 0xFB],
             // MP3 file with an ID3v2 container
-            [0x49, 0x44, 0x33]
+            [0x49, 0x44, 0x33],
+            // Other MP3 files (FF Fx and FF Ex – they may cause false-positives)
+            // Headers got from https://www.garykessler.net/library/file_sigs.html
+            [0xFF, 0xE0], [0xFF, 0xE1], [0xFF, 0xE2], [0xFF, 0xE3], [0xFF, 0xE4],
+            [0xFF, 0xE5], [0xFF, 0xE6], [0xFF, 0xE7], [0xFF, 0xE8], [0xFF, 0xE9],
```

Comment:
Isn't `x` referring to a hex number? So e.g. 0xEA - 0xEF are missing still ? Or am I thinking about this wrong?

Reply:
Hm, you're right, good point! Let me fix!

Reply:
Done!